include/dos_inc.h

```diff
@@ -123,6 +123,9 @@ bool DOS_ForceDuplicateEntry(uint16_t entry,uint16_t newentry);
 bool DOS_GetFileDate(uint16_t entry, uint16_t* otime, uint16_t* odate);
 bool DOS_SetFileDate(uint16_t entry, uint16_t ntime, uint16_t ndate);
 
+uint16_t DOS_GetCurPackedTime();
+uint16_t DOS_GetCurPackedDate();
```

Comment:
`DOS_GetCurrentPackedTime` 
`DOS_GetCurrentPackedDate` 

It's not like those few extra keystrokes cost money 😁

Reply:
(or at least use Curr instead of Cur)

Reply:
> `DOS_GetCurrentPackedTime` `DOS_GetCurrentPackedDate`
> 
> It's not like those few extra keystrokes cost money 😁

They don't cost money but they do cost space. You don't want function names to take up like half the page.

Reply:
Maybe use Curr then. I generally avoid abbreviations, except for the really common  ones, so maybe we can make an exception for "curr".

Reply:
All of the other words are fully expanded, so to be consistent it should be expanded too (we're only talking about four characters).

Otherwise, the pattern (of using full words) tells me `Cur` is probably an acronym or some other "full description" (which I don't yet know).

An alternative: `Now` is also commonly used in time code.

Suggest full expansion or collapse the others too: `DOS_GetCurPkdTm()`.. but that will probably mean we waste even more time painting the bike shed about four characters.

Reply:
`DOS_GetCurPkdTm` — now let's *not* do that 😅